src/test/java/com/auction/auction/bid/repositoryTests/AuctionRepositoryTest.java

```diff
@@ -0,0 +1,59 @@
+package com.auction.auction.bid.repositoryTests;
+import com.auction.auction.bid.model.Auction;
+import com.auction.auction.bid.repository.AuctionRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.junit.jupiter.api.extension.ExtendWith;
+import java.util.Date;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+@DataMongoTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class AuctionRepositoryTest {
+
+    @Autowired
+    private AuctionRepository auctionRepository;
+
+    @BeforeEach
+    void setUp() {
+        auctionRepository.deleteAll();
+    }
+
+    @Test
+    void testFindByOwner() {
+        Auction auction1 = new Auction();
+        auction1.setOwner("owner1");
+        auctionRepository.save(auction1);
+
+        Auction auction2 = new Auction();
+        auction2.setOwner("owner2");
+        auctionRepository.save(auction2);
```

Comment:
@clarammenezes you are using injection of real service but this is forbidden in scope of Unit tests because of next reasons:
1. Every time you running test - it will create new "test" records in DB (in real service no one allows it)
2.  Unit test is a test which testing only internal functionality of some particular method. So all your dependencies must me a Mocks but not real services. And you must mock behavior of all third-party services.
For example: if your method just call some method of another service, so all you need to do in scope of this test is just use something like:
1. Mock behavior: doRetorn("someListOfAuctions").when(auctionRepository).findByOwner("owner1");
2. Verify it was called: verify(auctionRepository).findByOwner('owner1');
3. Verify that you method returns exactly wat it got from auctionRepository: assertEquals("someListOfAuctions",  owner1Auctions )